package.json

```diff
@@ -123,7 +123,6 @@
     "cz-customizable": "^5.2.0",
     "eslint": "^4.2.0",
     "eslint-plugin-node": "^6.0.1",
-    "flow-bin": "^0.66.0",
     "flow-remove-types": "^1.2.1",
```

Comment:
Sorry to bother again! Just noticed that we can also remove `flow-remove-types`!

Reply:
Ah! No worries. I just ran [`npm-check`](https://www.npmjs.com/package/npm-check) to check for all unused deps and this is the list of unused deps returned in output:

Can you please confirm which ones to keep/remove?

`global`
`recast`
`uglifyjs-webpack-plugin`
`@commitlint/cli`
`@commitlint/prompt-cli`
`commitizen`
`conventional-changelog-lint-config-cz`
~`flow-remove-types`~
`husky`
`lint-staged`
`schema-utils`

Reply:
No one here

Reply:
I've cleaned `flow-bin`, `flow-remove-types` as suggested by @ematipico 
Also, updated PR description,